src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.img__link {
+  display: block;
+  height: 40px;
+}
+
+header {
```

Comment:
I think it would be better to add a class to a header. For instance `class="header"` and style it with class selector `.header {}`